packages/xstate-wallet/webpack.config.js

```diff
@@ -61,7 +68,16 @@ module.exports = {
       'NITRO_ADJUDICATOR_ADDRESS',
       'USE_INDEXED_DB'
     ]),
-    new HtmlWebpackPlugin({template: './index-template.html'})
+    new HtmlWebpackPlugin({template: './index-template.html'}),
+    new webpack.DefinePlugin({
+      // This is a bit messy, we should clean this up
```

Comment:
Another option is to use `VERSION` instead of `process.env.VERSION`. It might be more clear since `VERSION`, `COMMIT_HASH`, and `BRANCH` are not environment variables.